GitUI/CommandsDialogs/FormStash.cs

```diff
@@ -378,5 +378,14 @@ private void toolStripButton_customMessage_EnabledChanged(object sender, EventAr
                 StashMessage.ReadOnly = false;
             }
         }
+
+        private void View_KeyUp(object sender, KeyEventArgs e)
+        {
+            // Close Stash form with escape button while pointer focus is in FileViewer(diff view)
+            if (e.KeyCode == Keys.Escape)
+            {
+                Cancel.PerformClick();
```

Comment:
Is there stopping us from calling the form's `Close()` and closing the form?

Reply:
I dont think so, i was following the same pattern at FormCommit

Reply:
Hmm... thanks, `FormCommit` will need to get looked at.
Let's not add a new button here, just call `Close()`, if there is nothing stopping us from doing it.